### PR TITLE
fix: missing `expected-shas.json` from release

### PR DESCRIPTION
**Pull request by robertsLando**
Ref #285

### CODE REVIEW

**Comment by N0Cloud** [comment]
I can confirm, it fix the issue.

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   "files": [
     "lib-es5/*.js",
     "lib-es5/*.ts",
+    "lib-es5/*.json",
```
Comment (baparham) [inline package.json:14]: Is there an extra tab here?

Comment (robertsLando) [reply package.json:14]: Not sure, shouldn’t it be picked up by linter otherwise?

Comment (robertsLando) [reply package.json:14]: BTW I’m from phone now, just switched off my pc

**Comment by baparham** [comment]
It just looked like it was tabbed over when looking at it on my phone. I'll
check on my computer.

File: package.json (hunk 1 above)
Comment (baparham) [reply package.json:14]: it's fine, it was already indented, but in the diff on the github app everything _but_ this line was left justified 🙄

**Review by baparham — APPROVED** [review]

**Comment by baparham** [comment]
now we get to test out if the build job can overwrite binaries in v3.5 👍 then I'll tag 3.5.1 for @leerob to release

**Comment by N0Cloud** [comment]
Correct. 
It was a javascript/typescript error, don't think it need new binairies.

**Pull request merged**